### PR TITLE
Add graph based projects

### PR DESCRIPTION
**Pull request by hgcassiopeia**
#85

### CODE REVIEW

File: src/components/ProjectList/listOfProjects.js [hunk 1]
```diff
@@ -584,6 +584,41 @@ const projectList = [
...
+    githubLink: 'https://github.com/networkx/networkx',
+    description: 'Python library for studying graphs and networks',
+    tags: ['Python', 'graph', 'graph-based', 'graph visualization']
+  },
+  {
+    name: 'Igraph',
+    imageSrc: 'https://avatars3.githubusercontent.com/u/3735184?s=200&v=4',
+    githubLink: 'https://github.com/igraph/igraph',
+    description: 'Python Library for creating and manipulating graphs',
+    tags: ['Python', 'graph', 'graph-based', 'graph visualization']
```
Comment (sara-02) [inline src/components/ProjectList/listOfProjects.js]: Same tags as above can be used here as well. `Igraph` and `networkx` provide the same basic functionality.

File: src/components/ProjectList/listOfProjects.js [hunk 2]
```diff
@@ -584,6 +584,41 @@ const projectList = [
...
+  },
+  {
+    name: 'D3',
+    imageSrc: 'https://avatars1.githubusercontent.com/u/1562726?s=200&v=4',
+    githubLink: 'https://github.com/d3/d3',
+    description: 'JavaScript library for visualizing data using web standards',
+    tags: ['Javascript', 'graph', 'graph-based', 'graph visualization']
```
Comment (sara-02) [inline src/components/ProjectList/listOfProjects.js]: Few more tag could be `data visualization`, `dynamic plots`

File: src/components/ProjectList/listOfProjects.js [hunk 3]
```diff
@@ -584,6 +584,34 @@ const projectList = [
...
+    name: 'Plotly',
+    imageSrc: 'https://avatars3.githubusercontent.com/u/5997976?s=200&v=4',
+    githubLink: 'https://github.com/plotly',
+    description: 'A high-level Python declarative charting library',
+    tags: ['python', 'javascript', 'graph', 'graph-based', 'graph visualization']
+  },
+  {
+    name: 'NetworkX',
+    imageSrc: 'https://avatars3.githubusercontent.com/u/388785?s=200&v=4',
+    githubLink: 'https://github.com/networkx/networkx',
```
Comment (Roshanjossey) [inline src/components/ProjectList/listOfProjects.js]: Please add `/contribute` at the end of the URL

Comment (hgcassiopeia) [reply src/components/ProjectList/listOfProjects.js]: @Roshanjossey You mean 'Networkx/contribute', 'Igraph/contribute', 'D3/contribute' right?

Comment (hgcassiopeia) [reply src/components/ProjectList/listOfProjects.js]: @Roshanjossey Ok, I got it :sweat_smile: Now, I solved. But d3 don't have 'contribute'.

File: src/components/ProjectList/listOfProjects.js [hunk 4]
```diff
@@ -584,6 +584,34 @@ const projectList = [
...
+    name: 'Igraph',
+    imageSrc: 'https://avatars3.githubusercontent.com/u/3735184?s=200&v=4',
+    githubLink: 'https://github.com/igraph/igraph/contribute',
+    description: 'Python Library for creating and manipulating graphs',
+    tags: ['Python', 'graph', 'graph algorithms', 'complex networks']
+  },
+  {
+    name: 'D3',
+    imageSrc: 'https://avatars1.githubusercontent.com/u/1562726?s=200&v=4',
+    githubLink: 'https://github.com/d3/d3',
```
Comment (Roshanjossey) [inline src/components/ProjectList/listOfProjects.js]: @hgcassiopeia, just one more

Comment (hgcassiopeia) [reply src/components/ProjectList/listOfProjects.js]: @Roshanjossey Ok, I solved it. Please review again.

File: src/components/ProjectList/listOfProjects.js (hunk 3 above)
Comment (Roshanjossey) [reply src/components/ProjectList/listOfProjects.js]: @hgcassiopeia, I'm hoping that d3 will add more issue with `good first issues` flag and there will be issues in `/contribute` link in the future.

This is a change make by GitHub recently and we're moving forward with it.

**Comment by Roshanjossey** [comment]
@sara-02, thank you for reviewing this. I'm happy with the changes. Please merge if you're good with it.

I'll do a deploy soon.

**Pull request merged**